src/main/java/ReviewResponse.java

```diff
@@ -13,6 +16,8 @@ public class ReviewResponse {
     @Getter
     String okbb;
 
+    @Setter
+    String blabla;
 
```

Comment:
:warning: Method does not follow the Single Responsibility Principle (SRP) as it combines multiple responsibilities (comparison and returning boolean). (Severity: Major)